### DISCUSSION TITLE
Do you think that different programming font can have great influence on coding?

### DISCUSSION THREAD

**Discussion by flygoly** [rootfont/rootfont#29] [General]
In my opinion, when I'm coding with my favourite programming font, I will have better feeling and be more efficient.

**Comment by davidsneighbour** [comment]
A big part plays legibility, for instance:

- ligatures for computer "phrases" like "-->" etc
- readability of 0 vs O, I vs l, 1 vs I, x vs :times: etc
- having the text not look like a terminal font also helps with long programming work

**Reply by flygoly** [reply]
I agreed with you about that "having the text not look like a terminal font also helps with long programming work".
